### ISSUE TITLE
Cannot install package I can find with conan search

### ISSUE THREAD

**Issue by richard-vock** [conan-io/conan#5148]
(please note that there is a similar question -- the proposed answer (to clear the cache) is not helping here though)

General:
   OS: Arch Linux
   Conan Version: 1.15.0

Steps to reproduce:

    > mv ~/.conan{,_bck}
    > conan remote add stable https://api.bintray.com/conan/richard-vock/stable
    > conan search --remote=stable png++*

       Existing package recipes:

       png++/0.2.9-1@richard-vock/stable

    > cd /path/to/project/build
    > conan install .. build=missing

       ...
       png++/0.2.9-1@richard-vock/stable: Not found in local cache, looking in remotes...
       png++/0.2.9-1@richard-vock/stable: Trying with 'conan-center'...
       png++/0.2.9-1@richard-vock/stable: Trying with 'stable'...
       ERROR: Unable to find 'png++/0.2.9-1@richard-vock/stable' in remotes

conanfile.txt includes this line in [requires] block:

    png++/0.2.9-1@richard-vock/stable

Link to package is [here](https://bintray.com/richard-vock/stable/png%2B%2B%3Arichard-vock#files/richard-vock%2Fpng%2B%2B%2F0.2.9-1%2Fstable%2F0%2Fexport)

Is it the plus signs or did I miss something? Package was uploaded by myself using conan upload. It is not the only package from that repository (in fact all other available packages are used) -- it **is** the only one not being found though.

For completeness sake the complete conanfile.txt:

    [requires]
    fmt/5.2.1@bincrafters/stable
    eigen/3.3.5@conan/stable
    CLI11/1.6.2@cliutils/stable
    png++/0.2.9-1@richard-vock/stable
    CTRE/2.0@richard-vock/dev
    glfw/3.2.1@bincrafters/stable
    glew/2.1.0@bincrafters/stable
    baldr/0.1@richard-vock/dev

    [generators]
    cmake

    [options]

richard-vock/dev only exists in local cache.

Here is the png++ conanfile.py:

    from conans import ConanFile, CMake, tools

    class PngConan(ConanFile):
        name = "png++"
        version = "0.2.9-1"
        license = "BSD"
        url = "https://www.nongnu.org/pngpp/"
        description = "A C++ wrapper for the libpng library"
        generators = "cmake"
        requires = "libpng/1.6.36@bincrafters/stable"

        def source(self):
            tools.get(
                "http://download.savannah.nongnu.org/releases/pngpp/png++-0.2.9.tar.gz")

        def build(self):
            pass

        def package(self):
            self.copy("*.hpp", dst="include/png++", keep_path=False)

**Comment by uilianries** [comment]
Hi @richard-vock !

This error is known, it's a bug from Bintray: https://github.com/conan-io/conan/issues/4677

But if the original project is called pngpp why don't you use same name?

Also, I think @Artalus has a recipe for pngpp.

Regards!

**Comment by memsharded** [comment]
But that error #4677 was fixed long time ago, if it is still there, then a regression might have been introduced in Bintray recently.

**Comment by uilianries** [comment]
Oh, I just forgot that issue is closed! 

Anyway, I can confirm this error, I've reproduced here.

**Comment by niosHD** [comment]
Interesting, I was not aware of #4677 and this sounds very much like my issue in #5062. It would be great if someone from the Bintray team could have a quick look at the problem.

**Comment by Artalus** [comment]
@uilianries  @richard-vock Indeed I tinkered with png++ in [my repo](https://github.com/Artalus/conan-pngpp), it even [passed](https://travis-ci.org/Artalus/conan-pngpp) some building steps, but it kinda flew out of my attention span `¯\_(ツ)_/¯`

**Comment by richard-vock** [comment]
> But if the original project is called pngpp why don't you use same name?

It is not though, the project is called png++. It is true, that often pp is used as a suffix in the c++ world to work around just these kind of bugs.

I looked for existing packages a "long" time ago and found two, the one by Artalus (which had a problem I can't really remember now) and another one that I slightly modified for correct paths.

I used this package in local cache only for over a year now - the only thing that changed is that I uploaded it (so it being a bintray issue makes perfect sense). For now I have my workaround - let's see if there's really a regression...

Best, Richard

**Comment by danimtb** [comment]
I have reproduced this issue with Bintray uploading a recipe and package with the same reference `png++/0.2.9-1@richard-vock/stable`.

In my case the search of packages does not work (it returns html output) and the install command does not find the package:

```
$ conan install png++/0.2.9-1@richard-vock/stable
DEBUG :conan_api.py   [202]: INIT: Using config 'C:\Users\danimtb\.conan\conan.conf' [2019-05-29 11:22:02,066]
DEBUG :tracer.py      [151]: CONAN_API: install_reference(settings=None,options=None,env=None,remote_name=None,verify=None,manifests=None,manifests_interactive=None,build=None,profile_names=None,update=False,generators=None,install_folder=None) [2019-05-29 11:22:02,073]
DEBUG :profile_loader.py[104]: PROFILE LOAD: C:\Users\danimtb\.conan\profiles\default [2019-05-29 11:22:02,078]
INFO  :settings_preprocessor.py[58]: Setting 'compiler.runtime' not declared, automatically adjusted to 'MD' [2019-05-29 11:22:02,117]
png++/0.2.9-1@richard-vock/stable: Not found in local cache, looking in remotes...
png++/0.2.9-1@richard-vock/stable: Trying with 'helloworld'...
DEBUG :rest_client_common.py[109]: REST: ping: https://api.bintray.com/conan/danimtb/helloworld/v1/ping [2019-05-29 11:22:03,558]
DEBUG :rest_client_common.py[135]: REST: get: https://api.bintray.com/conan/danimtb/helloworld/v2/conans/png++/0.2.9-1/richard-vock/stable/latest [2019-05-29 11:22:04,170]
DEBUG :rest_client_common.py[37]: REST ERROR: <class 'conans.errors.NotFoundException'> [2019-05-29 11:22:04,353]
ERROR: Unable to find 'png++/0.2.9-1@richard-vock/stable' in remotes
```

The package was uploaded here: https://bintray.com/danimtb/helloworld/png%2B%2B%3Arichard-vock

Thanks for reporting!

**Comment by danimtb** [comment]
As @niosHD reported in this issue  #5062, this seems to be a duplicated of that one. Please follow the conversation there.

**Comment by danimtb** [comment]
Marked as fixed in #5062
